jenkins/gce-custom-images/create_gce_image.sh

```diff
@@ -65,5 +65,10 @@ docker run -it --rm -v "$SOURCE_DIR":/gce-custom-images \
   -var:installation_script_name prepare_gce_image.sh \
   /gce-custom-images/gce_image.wf.json
 
+gcloud beta compute images add-iam-policy-binding \
+    projects/$GOOGLE_PROJECT/global/images/${OUTPUT_IMAGE_NAME} \
+    --member='allAuthenticatedUsers' \
+    --role='roles/compute.imageUser'
```

Comment:
@Qi77Qi I'm guessing we hadn't run this for `staging` which caused the UI integration test failure?

Reply:
it was never run becuz the image was generated using this script..this is not env specific